<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDialogClass</class>
 <widget class="QMainWindow" name="FormDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormDialog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="cForm">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>695</y>
      <width>131</width>
      <height>126</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>870</width>
      <height>830</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>868</width>
       <height>828</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
